cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/template/etl/batch/ETLStreamConversionTest.java

```diff
@@ -64,13 +67,32 @@
     Schema.Field.of("price", Schema.of(Schema.Type.DOUBLE)));
 
   @Test
-  public void testStreamConversion() throws Exception {
+  public void testStreamConversionTPFSParquetSink() throws Exception {
```

Comment:
seems like these two methods are exactly the same, except for the sinkType. Worth extracting into common method, to reduce code redundancy?